### ISSUE TITLE
Hunt function prints Malloc errors on screen

### ISSUE THREAD

**Issue by roberthawdon** [roberthawdon/dfshow#110]
Using the hunt function can cause memory allocation errors.

```
show(9098,0x10ffd95c0) malloc: *** set a breakpoint in malloc_error_break to debug
show(9098,0x10ffd95c0) malloc: can't allocate region
*** mach_vm_map(size=18446744071562067968) failed (error code=3)
```

**Comment by roberthawdon** [comment]
Hoping this is fixed in 0.8.2, if not, will reopen.

**Comment by roberthawdon** [comment]
Issue is still occurring, seems to be MacOS specific

**Comment by roberthawdon** [comment]
This also happens with the Find function in SF

**Comment by roberthawdon** [comment]
Adding help wanted flag as this seems to be some very weird MacOS edge case.

**Comment by roberthawdon** [comment]
The latest code optimisations should have now resolved this in the upcoming v0.9.4 release. I will close for now. Again if this crops up again, we'll reopen it.
